subprojects/packagefiles/ogg/src/meson.build

```diff
@@ -4,6 +4,7 @@ libogg = library('ogg',
   sources,
   include_directories : incdir,
   vs_module_defs: '../win32/ogg.def',
+  install: true,
```

Comment:
While we are at it, the official build system installs this with a soname "0" and a full version 0.8.5